libcxx/include/__config_site.in

```diff
@@ -45,6 +45,7 @@
 #endif
 
 @_LIBCPP_ABI_DEFINES@
+@_LIBCPP_SANITIZER_DEFINES@
```

Comment:
You can remove this and instead use `#cmakedefine _LIBCPP_INSTRUMENTED_WITH_ASAN` above, right below `_LIBCPP_HAS_NO_TIME_ZONE_DATABASE`.